templates/_templating_config.json

```
@@ -23,5 +23,8 @@
   "benchmarks/custom_bms/README.md": {
     "iris": "benchmarks/custom_bms/README.md",
     "iris-esmf-regrid": "benchmarks/custom_bms/README.md"
+  },
+  "linkcheck/ci-linkchecks.yml": {
```

Offline conversation with @ESadek-MO: we favour mirroring the real structure as much as possible. Helps people when they're copying a template for the first time, and reduces ambiguity when people are creating templates for the first time.

```suggestion
  ".github/workflows/ci-linkchecks.yml": {
```

---

I hope https://github.com/SciTools/.github/pull/68/commits/fd9c3bb4275cb4befceadf99fd425f5c6d0da2ac does the job.

As discussed, I'm avoiding using an actual `.github` folder as maybe that can be just too confusing.